lib/jekyll/jekyll-feed.rb

```diff
@@ -18,6 +18,12 @@ def generate(site)
 
     private
 
+    # Matches all whitespace that follows
+    #   1. A '>', which closes an XML tag or
+    #   2. A '}', which closes a Liquid tag
+    # We will strip all of this whitespace to minify the template
+    MINIFY_REGEX = %r!(?<=>|})\s+!
```

Comment:
This _could_ be changed to `%r!(?<=[>}])\s+!`

I don't know if that would be any better or worse 🐳